### DISCUSSION TITLE
Search Result Preview for URL file

### DISCUSSION THREAD

**Discussion by mugiwara23** [microsoft-search/pnp-modern-search#2013] [Q&A]
Is there a fix on the Search Result preview for URL file. I'm currently using 4.5.4

![151196601-900055e0-2dcd-45a4-91d1-296344fd187a](https://user-images.githubusercontent.com/93260954/162716202-238b8947-268e-4604-877c-f8b34a7ead1e.png)

Thank  you

**Comment by wobba** [comment]
You can use v3 which loads the data in the .url file and grabs the correct URL.. or live with that experience and click "Open" which should do the right thing. Boils down to SharePoint not doing the right thing for these files and no easy way to get around that in search.

**Reply by mugiwara23** [reply]
Thanks Wobba. 

So this function was deleted in V4 right? I really want to use V4 because it runs smoothly on our document library. 
Is there a chance that it could be included on the next release?

**Reply by wobba** [reply]
Not deleted, just not implemented yet.
